README.md

```diff
@@ -1,86 +1,56 @@
 
-The Cozy Drive for Desktop application allows to sync the files stored in your
-Cozy with your laptop and/or your desktop. It replicates your files on your
-hard drive and apply changes you made on them on other synced devices and on
-your online Cozy.
+-   [Setting up a development environment](doc/developer/setup.md)
+-   [Other developer documentation](doc/developer/)
 
 ## Disclaimer
 
 :warning: Cozy Drive for Desktop is still in beta version. Backup your data before using it!
 
+## Concerning the old Cozy
 
-## Installing & using the application
-
-See the [official documentation](https://docs.cozy.io/en/sync/desktop/).
-
-
-## Advanced use cases
-
-- [Installing and using the command-line version](doc/usage/cli.md)
-- [Synchronizing multiple directories](doc/usage/multi_directory_setup.md)
-- [Ignore files](doc/usage/ignore_files.md)
-- [Limitations](doc/usage/limitations.md)
-
-
-## Cozy v2 users
-
-The new Cozy Drive for Desktop application targets the new Cozy stack v3 only.
-
-Maintenance for the old cozy-desktop GUI / CLI (targeting Cozy v2) is still
-occuring here. Support requests are definitely welcome, although we're quite
-busy with the v3 release.
-
-
-## GNU/Linux & BSD users
-
-The new Cozy Drive for Desktop application should work on Linux and
-BSD flavors. It has been tested in development on Ubuntu/Gnome, however we do not provide init scripts, service files or packages yet, so you'll have to build and install it by hand. Any contribution in this direction will be appreciated.
-Packages for major GNU/Linux distributions should come up.
-
-
-## Contributing
-
-- [Setting up a development environment](doc/developer/setup.md)
-- [Other developer documentation](doc/developer/)
-
+The new Cozy Drive for Desktop application targets the new [Cozy Stack](https://github.com/cozy/cozy-stack) only.
 
-## Get in touch
+Maintenance for the old cozy-desktop GUI / CLI (targeting the old Cozy) is still occuring here.
+Support requests are definitely welcome, although we're quite busy with the new release.
 
-The best way to ask for help is through the form embedded in the application.
+## Feedback
 
 You can also:
 
-* Discuss on the [Forum](https://forum.cozy.io)
-* Report issues on the [Github repository](https://github.com/cozy-labs/cozy-desktop/)
-* Chat with us on the [IRC channel](irc://irc.freenode.net/cozycloud)
-* Mention us on [Twitter](https://twitter.com/mycozycloud)
+-   Discuss on the [Forum](https://forum.cozy.io)
+-   Report issues on the [Github repository](https://github.com/cozy-labs/cozy-desktop/issues)
+-   Chat with us on the [IRC channel](irc://irc.freenode.net/cozycloud)
+-   Mention us on [Twitter](https://twitter.com/mycozycloud)
 
-The current maintainers of Cozy Drive for Desktop are @sebn & @aenario, feel
-free to say hello!
+The current maintainers of Cozy Drive for Desktop are @sebn & @aenario, feel free to say hello!
```

Comment:
I will update this to use a github team with listed members, instead of individual members.